lnrpc/routerrpc/routing_config.go

```diff
@@ -47,4 +35,51 @@ type RoutingConfig struct {
 	// McFlushInterval defines the timer interval to use to flush mission
 	// control state to the DB.
 	McFlushInterval time.Duration `long:"mcflushinterval" description:"the timer interval to use to flush mission control state to the DB"`
+
+	// AprioriConfig defines parameters for the apriori probability.
+	AprioriConfig *AprioriConfig `group:"apriori" namespace:"apriori" description:"configuration for the apriori pathfinding probability estimator"`
+
+	// BimodalConfig defines parameters for the bimodal probability.
+	BimodalConfig *BimodalConfig `group:"bimodal" namespace:"bimodal" description:"configuration for the bimodal pathfinding probability estimator"`
```

Comment:
You could first do a commit that restructures the config for apriori only, so that the real addition of bimodal stands out more in its own commit.

Reply:
See the other comment